WikiDrips/WikiTableViewController.swift

```diff
@@ -193,10 +223,20 @@ extension WikiTableViewController: UITableViewDataSourcePrefetching {
     func tableView(_ tableView: UITableView, prefetchRowsAt indexPaths: [IndexPath]) {
         indexPaths.forEach {
             checkImage(forItemAtIndex: $0)
+            if shouldGetNextPage(indexPath: $0) {
+                pendingSearch = true
```

Comment:
Seems like this wouldn't be necessary since it's being set in `createSearchRequest`